modules/nixos/general/keyboard.nix

```diff
@@ -53,7 +53,7 @@ with lib;
     i18n.consoleKeyMap = let layout = layouts."${builtins.head cfg.layouts}"; in
                          if layout.variant == ""
                          then layout.layout
-                         else layout.variant;
+                         else (if layout.variant == "bepo" then "fr-bepo" else layout.variant);
```

Comment:
Wouldn't it be better to split `layout` into `consoleLayout` and `xLayout`?

Reply:
Yes perhaps, but maybe we should keep that for a future PR when we will be improving the keyboard layouts in nvim/tmu/i3/...

Reply:
Fair enough.